client/js/components/user/orgaDataEntry/OrganisationDataForm.vue

```diff
@@ -1,15 +1,34 @@
 <template>
-  <div>
-    <fieldset class="w-3/4">
-      <legend class="font-size-large weight--normal mb-3">
+  <form
+    :class="prefixClass('u-mt')"
+    :action="Routing.generate('DemosPlan_orga_edit_save', { orgaId: organisation.id })"
+    @submit="precheckSubmit"
+    method="post"
+    data-dp-validate="orgadata">
+    <input
+      data-cy="editOrga:organisationId"
+      type="hidden"
+      name="organisation_ident"
+      :value="organisation.id">
+    <input
+      data-cy="editOrga:addressIdent"
+      type="hidden"
+      name="address_ident"
+      :value="organisation.addressId">
+    <input
+      type="hidden"
+      name="_token"
+      :value="csrfToken">
+    <fieldset :class="prefixClass('w-3/4')">
+      <legend :class="prefixClass('font-size-large weight--normal mb-3')">
         {{ Translator.trans('organisation.data') }}
       </legend>
 
         <!-- Name -->
         <dp-input
           id="orga_name"
           v-model="organisation.name"
-          class="mb-2"
+          :class="prefixClass('mb-2')"
           data-cy="organisationData:name"
```

Comment:
Do we still need these prefixClasses?

Reply:
Yes. we should keep that functionality. ;-)